### DISCUSSION TITLE
favourites

### DISCUSSION THREAD

**Discussion by steveparkerdj** [DISTRHO/Cardinal#278] [General]
I don't know if this is a bug to be reported but favourites modules sometimes show up other times don't. 
Mac Catalina Au with Ableton live.

Thanks

**Comment by dromer** [comment]
Favorites are saved as part of the plugin state (iirc). this means that you can only see them as part of your current instance inside the current DAW session.

There is no global folder where such data is stored between all Cardinal sessions. It's literally part of the DAW session and afaik only for that specific instance (if you have multiple tracks with Cardinal, each will have their own plugin state).

**Comment by falkTX** [comment]
What @dromer said.
Cardinal is a plugin and currently has zero global state, it does not write any config files to disk, the entire state is saved as part of the host/daw project.
If there is a module that saves some preferences or related things to disk, consider that a bug.

So basically, this is by design, and working as intended.
